public/javascripts/chatty.js

```diff
@@ -21,6 +25,19 @@ let publisher = OT.initPublisher('container', publisherOptions, (err) => {
   }
 });
 
+publisher.on({
+  accessDenied: (event) => {
+    const errorBox = document.getElementById('video-container');
+    errorBox.innerHTML = `Chatty requires access to your microphone and camera.  
```

Comment:
innerHTML does a complete re-render, how about creating a text node and then adding that as a child?